packages/features/eventtypes/components/OrganizationEventTypeFilter.tsx

```diff
@@ -0,0 +1,118 @@
+import { useSession } from "next-auth/react";
```

Comment:
followup: It can be replaced with [TeamsFilter ](https://github.com/calcom/cal.com/pull/9417/files#diff-08a37fc124270dd05eb90e0e85aa8d77cff174f1bfcde23e84c371ced895570b)component